### PR TITLE
Add Sprite support

### PR DESCRIPTION
**Pull request by KaiVolland**
## Description
This updates geostyler-style and geostyler-openlayers-parser to their latest versions and adds support for the geostyler-style `Sprite`.

It adds a small Fieldset to the ImageInput which is collapsed by default.

image

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

### CODE REVIEW

File: src/Component/Symbolizer/Field/ImageField/ImageField.tsx [hunk 1]
```diff
@@ -38,10 +37,18 @@ import { IconLibrary } from '../../IconSelector/IconSelector';
...
+
+import './ImageField.less';
+import { FieldSet } from '../../../FieldSet/FieldSet';
+import NumberExpressionInput from '../../../ExpressionInput/NumberExpressionInput/NumberExpressionInput';
 
 export interface ImageFieldProps {
   /** The tooltip label for the iconLibraries button. */
   tooltipLabel?: string;
+  /** The tooltip label for the sprite button. */
+  spriteTooltipLabel?: string;
```
Comment (jansule) [inline src/Component/Symbolizer/Field/ImageField/ImageField.tsx:51]: Could you add the `spriteTooltipLabel` to the i18n files?

**Review by jansule — COMMENTED** [review]
Nice work @KaiVolland! Could you also adjust the iconLibrary to support sprites, so that all image related inputs still work together? Also, could you add the support for sprites to the composition context, so that it can be en-/disabled for different scenarios?

**Comment by KaiVolland** [comment]
I added an issue for the extend UI: https://github.com/geostyler/geostyler/issues/2390

File: src/Component/Symbolizer/Field/ImageField/ImageField.tsx (hunk 1 above)
Comment (KaiVolland) [reply src/Component/Symbolizer/Field/ImageField/ImageField.tsx:51]: Solved with [5b68e8d](https://github.com/geostyler/geostyler/pull/2389/commits/5b68e8d8bbb88610da38d42f5e02083119e0c0fc)

**Pull request merged**